pkgs/tools/filesystems/catcli/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonApplication
+, docopt, anytree 
+}:
+
+buildPythonApplication rec {
+
+  pname = "catcli";
+  version = "v0.5.13";
+  
+  src = fetchFromGitHub {
+    owner = "deadc0de6";
+    repo = pname;
+    rev = version;
+    sha256 = "04mrkqmm2c8fhi1h1hddc4yh3dqhcvkmcwzhj0ggn34v7wavgb5i";
+  };
+
+  propagatedBuildInputs = [ docopt anytree ];
+
+  postPatch = '' patchShebangs . '';
+
+  meta = with stdenv.lib; {
+    description = "The command line catalog tool for your offline data";
+    homepage = "https://github.com/deadc0de6/catcli";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ petersjt014 ];
+    platforms = with platforms; [ x86_64-linux i686-linux ];
```

it builds on my pi...
```suggestion
    platforms = platforms.all;
```

---

Just tried and it builds natively on mine too.
Unfortuantely, the `raspberryPi` cross-build target still doesn't work (and I haven't tested it on anything else) so I'm hesitant to use `all` for now.

---

For anyone wondering this is the build log for `raspberryPi`: http://0x0.st/iu-6.txt